mkosi/__init__.py

```diff
@@ -286,13 +318,12 @@ def __str__(self) -> str:
         return self.name
 
     @classmethod
-    def from_string(cls, name: str) -> "OutputFormat":
+    def Format(cls, name: str) -> "OutputFormat":
```

Comment:
The uppercase method name is a bit unusual, since uppercase things are usually classes. How about `format_from_string` or `get_format` or just keeping it `from_string`?

Reply:
Please see the commit message: "Format" is the same as the option in the config file, so we get a nice message:
```
    mkosi: error: argument -t/--format: invalid Format value: 'cpioa'
```

"invalid from_string/format_from_string/get_format" are not nearly as nice...

Reply:
Ah, I didn't get that. I see the point. `format` in lowercase could make this look, like it had a (stringlike) format method, but otherwise I don't think it would clash with anything.

Reply:
I'd prefer we stick to the lowercase naming style here. The message is nice but it wouldn't be consistent across all options so if we do this I'd prefer it to be all or nothing.